### PR TITLE
Explain what is the Symfony Core team

### PR DESCRIPTION
**Pull request by javiereguiluz**
We have an article explaining the rules that govern the Symfony Core ... but we don't explain anywhere what the Symfony Core team is.

A review from a native English speaker (@weaverryan ?) will be greatly appreciated. Thanks!

### CODE REVIEW

File: contributing/code/core_team.rst [hunk 1]
```diff
@@ -1,7 +1,15 @@
 Symfony Core Team
 =================
 
-This document states the rules that govern the Symfony Core group. These rules
+The **Symfony Core** team is the group of developers that determine the
+direction and evolution of the Symfony project. Their votes rule if the
+features proposed by the community are approved or rejected.
+
+All the Symfony Core members are longtime contributors with a solid technical
```
Comment (xabbuh) [inline contributing/code/core_team.rst]: long-time?

File: contributing/code/core_team.rst [hunk 2]
```diff
@@ -1,7 +1,15 @@
 Symfony Core Team
 =================
 
-This document states the rules that govern the Symfony Core group. These rules
+The **Symfony Core** team is the group of developers that determine the
+direction and evolution of the Symfony project. Their votes rule if the
+features proposed by the community are approved or rejected.
```
Comment (HeahDude) [inline contributing/code/core_team.rst]: patches and features?

**Comment by HeahDude** [comment]
👍

**Comment by weaverryan** [comment]
👍 and merged!

**Pull request merged**